Purpose: Get insserv building on m68k, dropping some problematic
         alignments (error: requested alignment is not a power of 2)
Fixes:   #493637
Status:  Not yet submitted upstream.
---
--- a/insserv.c
+++ b/insserv.c
@@ -53,6 +53,10 @@
 #endif /* SUSE */
 #include "listing.h"
 
+#ifdef __m68k__ /* Fix #493637 */
+#  define aligned(a)
+#endif
+
 #if defined _XOPEN_SOURCE && (_XOPEN_SOURCE - 0) >= 600
 # ifndef POSIX_FADV_SEQUENTIAL
 #  define posix_fadvise(fd, off, len, adv)  (-1)
